feature-libs/customer-ticketing/components/customer-ticketing-create/customer-ticketing-create.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Component,
+  ElementRef,
+  OnDestroy,
+  ViewChild,
+  ViewContainerRef,
+} from '@angular/core';
+import { LaunchDialogService, LAUNCH_CALLER } from '@spartacus/storefront';
+import { Subscription } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-customer-ticketing-add',
```

Comment:
the selector name is different from the component name

usually when you create a component through ng cli, they're uniform/similar. So I believe this should have been cx-customer-ticketing-create